hermes/bin/src/logger.rs

```diff
@@ -0,0 +1,75 @@
+//! Setup for logging for the service.
+
+use clap::ValueEnum;
+use tracing::{level_filters::LevelFilter, subscriber::SetGlobalDefaultError};
+use tracing_subscriber::{
+    fmt::{format::FmtSpan, time},
+    FmtSubscriber,
+};
+
+/// All valid logging levels.
+#[derive(ValueEnum, Clone, Copy)]
+pub(crate) enum LogLevel {
+    /// Errors
+    Error,
+    /// Warnings
+    Warn,
+    /// Informational Messages
+    Info,
+    /// Debug messages
+    Debug,
+    /// Tracing
+    Trace,
+}
+
+impl From<&str> for LogLevel {
+    fn from(s: &str) -> Self {
+        match s.to_lowercase().as_str() {
+            "error" => LogLevel::Error,
```

Comment:
```suggestion
            "error" => LogLevel::Info // Highest log level one can choose is Info, so ensure it here.
```

Reply:
apply this change to force all conversion of `&str` and `runtime_extensions::bindings::hermes::logging::api::Level`